waspc/data/Generator/templates/react-app/src/auth/pages/DefaultPageLoader.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+// TODO: Should I take this out of auth/, into more general place?
```

Comment:
The question is also, do we want to go with
`app.pageLoader` in `main.wasp`, or `app.auth.pageLoader`.
The thing is, concept of page loader could be generic. While resources are loading for a page that block its showing, we use page loader.
However, we currently use that only to fetch auth user.
So we could also go with it being more specialized.
I am not sure. I think I would immediately go general from the DX side, while internally we will apply it only in the auth user situation. Which means `app.pageLoader`, not `app.auth.pageLoader`. And also, this file here, would go some levels up.